tests/console/yast2_vnc.pm

```diff
@@ -46,13 +46,13 @@ sub configure_remote_admin {
 
 sub check_service_listening {
     my $cmd_check_port = $is_older_product ? 'netstat' : 'ss -tln | grep -E LISTEN.*:5901';
-    if (script_run($cmd_check_port)) {
-        record_soft_failure 'boo#1088646 - service vncmanager is not started automatically';
-        systemctl('status vncmanager', expect_false => 1);
-        systemctl('restart vncmanager');
-        systemctl('status vncmanager');
```

Comment:
Tbh, I would still add this check too, don't know why we have it only in the soft-failure case. WDYT?

Reply:
The ancient code was soft-faliing without verifying first that the service is really off, so the bug being closed. So my first idea was also to check fthe status eg
```
        return if (!script_run(systemctl is-active vncmanager);
        record_soft_failure 'boo#1088646 - service vncmanager is not started automatically';
        systemctl('restart vncmanager');
        systemctl('status vncmanager');
```
 but since the bug is long closed I think we should just fail if something is not as expected.

Reply:
If you don't object, I would add `systemctl('status vncmanager');` to current test, so we are sure not only that port is open, as it can be opened by other service, for instance.

Reply:
Ah, it's already there now, thanks!

Reply:
You mean like at line 55 :thinking: ?